inference_perf/datagen/random_datagen.py

```diff
@@ -107,7 +107,9 @@ def is_io_distribution_supported(self) -> bool:
     def is_shared_prefix_supported(self) -> bool:
         return False
 
-    def get_request(self, n: int) -> InferenceAPIData:
```

Comment:
This is causing test error https://github.com/kubernetes-sigs/inference-perf/actions/runs/19481712318/job/55976961489?pr=257